{"uuid": "b915ae1d-1802-4827-be1f-ff2b3292abc6", "children": ["43dbab6b-ba35-4ca9-beab-68803deb8ce7", "f6fb08c2-34a2-42a5-9787-9f514c92c21f", "de32a1ff-66f7-4cad-8990-d9af1c6eefb5", "111e41a2-5891-4ef3-996b-0090fc195012", "9861d4d2-dfff-4fdb-8052-8ed3302b0e38", "1d3e55f5-7b6c-4260-99b0-3884bbd811d0"], "befores": [{"name": "setup_profiles", "status": "failed", "statusDetails": {"message": "Failed: AP latest and active are different\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/interOp/conftest.py\", line 228, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_ref,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 571, in setup_profiles\n    pytest.fail(\"AP latest and active are different\")\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/outcomes.py\", line 153, in fail\n    raise Failed(msg=msg, pytrace=pytrace)\n"}, "attachments": [{"name": "Active Config: ", "source": "7f6e8d52-2cb9-409b-9f59-0b44608dca62-attachment.attach"}, {"name": "Latest Config: ", "source": "ad6dd0c0-b03e-4a60-abe7-f9db1ba12b8b-attachment.attach"}, {"name": "FAILURE: AP Logs: ", "source": "b3a0a941-dab4-429a-9521-30e16ecd4b34-attachment.attach"}], "start": 1639085087161, "stop": 1639085409203}], "start": 1639085087161, "stop": 1639085411082}